### PR TITLE
Router refactor

### PR DESCRIPTION
**Pull request by wsakolski**
small router refactor and simple navbar styling

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  useParams,
+} from 'react-router-dom'
+import { Test } from './components/test.component'
+import { Home } from './components/home.component'
+import { About } from './components/about.component'
+import WhoAreYou from './pages/WhoAreYou'
+import { Navbar } from './components/navbar.component'
+
+function App() {
+  return (
```
Comment (sylwiaszymanska) [inline src/App.jsx:16]: You don't need {} return () here ;)
Can be replaced with `const App = () => ( ...components)`

File: src/components/navbar.component.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    justify-content: flex-start;
+    & *:not(:last-child) {
+      margin-right: 30px;
+    }
+  }
+`
+export const Navbar = () => (
+  <NavbarContainer>
+    <div>
+      <div>LOGO</div>
```
Comment (sylwiaszymanska) [inline src/components/navbar.component.jsx:36]: We can change that to custom component, e.g. `const LinkWithLogo = styled.a` styles here`

File: src/components/navbar.component.jsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    align-items: center;
+    justify-content: flex-start;
+    & *:not(:last-child) {
+      margin-right: 30px;
+    }
+  }
+`
+export const Navbar = () => (
+  <NavbarContainer>
+    <div>
```
Comment (sylwiaszymanska) [inline src/components/navbar.component.jsx:35]: Isn't this container redundant? Why do you need NavbarContainer and div inside (they have almost the same styles)?

File: src/App.jsx (hunk 1 above)
Comment (wsakolski) [reply src/App.jsx:16]: Oki :) Done!

File: src/components/navbar.component.jsx (hunk 2 above)
Comment (wsakolski) [reply src/components/navbar.component.jsx:36]: Already done!

**Pull request merged**